src/cli.ts

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 // tslint:disable-next-line
-':' //# -*- mode: js -*-; exec /usr/bin/env node --max-old-space-size=9000 --stack-size=42000 "$0" "$@"
+':'; //# -*- mode: js -*-; exec /usr/bin/env node --max-old-space-size=9000 --stack-size=42000 "$0" "$@"
```

Comment:
I'm not sure about that? What the reason behind?

Reply:
sorry might be done by the js extension will revert it back